java-client/src/test/java/org/opensearch/client/opensearch/core/SearchRequestTest.java

```diff
@@ -32,6 +32,24 @@ public void ext() {
         assertEquals("{\"ext\":{\"similarity\":{\"fields\":[\"name\"]}}}", toJson(request));
     }
 
+    @Test
+    public void phaseTook() {
+        SearchRequest request = new SearchRequest.Builder().phaseTook(true).build();
+
+        assertEquals("{}", toJson(request));
```

Comment:
Shouldn't phase_took` appear somewhere in the JSON?

Reply:
`phase_took` and `pipeline` are query parameters whereas `search_after` and `ext` are in the request body. I think that's what toJson is testing